content/design-guidelines/backgrounds.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Backgrounds
+---
+
+## Backgrounds
+
+Background colors should be limited to the colors provided in the template.
+
+![Screenshot 2019-04-02 13 55 51](https://user-images.githubusercontent.com/10384315/55435593-0a97ab00-554f-11e9-83b1-74aa14dd7411.png)
```

Comment:
Just noticed there's no alt text here, can you check the rest of the images to make sure they have one?